clouddq/lib.py

```diff
@@ -131,13 +131,27 @@ def create_rule_binding_view_model(
         high_watermark_filter_exists=high_watermark_filter_exists,
         bigquery_client=bigquery_client,
     )
-
-    sql_string = template.render(configs)
-    failed_records_sql_string = failed_records_template.render(configs)
-    configs.update({"generated_sql_string": sql_string})
-    configs.update({"failed_records_sql_string": failed_records_sql_string})
+    rule_configs_dict = configs.get("configs").get("rule_configs_dict")
+    generated_sql_string_dict = dict()
+    for rule_id, rule_configs in rule_configs_dict.items():
+        configs.update({"rule_type": rule_configs.get("rule_type")})
+        configs.update({"rule_id": rule_id})
+        failed_records_sql_string = failed_records_template.render(configs)
+        existing_configs = configs.get("configs")
+        existing_configs[
+            f"{rule_binding_id}_{rule_id}_failed_records_sql_string"
+        ] = failed_records_sql_string
+        configs.update({"configs": existing_configs})
+        sql_string = template.render(configs)
+        generated_sql_string_dict[
+            f"{rule_binding_id}_generated_sql_string"
+        ] = sql_string
```

Comment:
Should these lines be outside of for-loop? Otherwise, we are just keep overriding `generated_sql_string_dict[f"{rule_binding_id}_generated_sql_string"]`?